docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -540,64 +547,67 @@ spec:
   - pre-adoption-validation
 EOF
 ----
-+
-Wait for the validation to finish.
+
+.. Wait for the validation to finish.
 
 .. Confirm that all the Ansible EE pods reach a `Completed` status:
 +
 ----
 # watching the pods
-watch oc get pod -l app=openstackansibleee
+$ watch oc get pod -l app=openstackansibleee
 ----
 +
 ----
 # following the ansible logs with:
-oc logs -l app=openstackansibleee -f --max-log-requests 20
+$ oc logs -l app=openstackansibleee -f --max-log-requests 20
 ----
 
 .. Wait for the deployment to reach the `Ready` status:
 +
 ----
-oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
+$ oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
 ----
++
+[IMPORTANT]
+====
+If any openstack-pre-adoption validations fail, you must determine
+which ones were unsuccessful based on the Ansible logs, and then try the
+following troubleshooting options for each case:
 
-. If any openstack-pre-adoption validations fail, you must first determine
-which ones were unsuccessful based on the ansible logs and then follow the
-instructions below for each case:
-
-* if the hostname validation failed then check that the hostname of the EDPM
-node is correctly listed in the `OpenStackDataPlaneNodeSet`
+* If the hostname validation failed, check that the hostname of the data plane
+node is correctly listed in the `OpenStackDataPlaneNodeSet` CR.
 
-* if the kernel argument check failed then make sure that the
-`OpenStackDataPlaneNodeSet` has the same kernel argument configuration in
-`edpm_kernel_args` and `edpm_kernel_hugepages` variables than what is used in
-the 17 node.
+* If the kernel argument check failed, ensure that the
+`OpenStackDataPlaneNodeSet` CR has the same kernel argument configuration in the
+`edpm_kernel_args` and `edpm_kernel_hugepages` variables as the one used in
+the {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver} node.
 
-* if the tuned profile check failed then make sure that the
-`edpm_tuned_profile` variable in the `OpenStackDataPlaneNodeSet` is configured
-to use the same profile as set on the (source) OSP 17 node.
+* If the tuned profile check failed, ensure that the
+`edpm_tuned_profile` variable in the `OpenStackDataPlaneNodeSet` CR is configured
+to use the same profile as the one set on the {OpenStackShort} {rhos_prev_ver} node.
+====
 
-. Remove leftover {OpenStackPreviousInstaller} services
```

Comment:
is this still part of the pre-validation?

If not, maybe we could have separate sections like:

1 Pre-validating
2 Troubleshooting failed pre-validating
3 Validating (or whatever comes after pre-validating, completing the adoption or whatever)

Reply:
yes, it is not a pre-validataion. It is between 2 and 3